geode-web-management/src/main/java/org/apache/geode/management/internal/rest/ManagementLoggingFilter.java

```diff
@@ -31,21 +31,12 @@
 
 public class ManagementLoggingFilter extends OncePerRequestFilter {
 
-  // Because someone is going to want to disable this.
-  private static final Boolean ENABLE_REQUEST_LOGGING =
-      Boolean.parseBoolean(System.getProperty("geode.management.request.logging", "true"));
-
   private static int MAX_PAYLOAD_LENGTH = 10000;
 
   @Override
   protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response,
       FilterChain filterChain) throws ServletException, IOException {
 
-    if (!ENABLE_REQUEST_LOGGING) {
-      filterChain.doFilter(request, response);
-      return;
-    }
-
```

Comment:
This change seems to have an adverse performance impact.  Even if debug logging is not enabled, we will now always wrap and capture the request and response.  Before, the wrapping and capturing was able to be skipped.  Why not keep the same logic as before, just change the test to from system-property check to isDebugEnabled check?

Reply:
Yes, that's a good point, I'll make that change.